filesystem/src/main/java/org/frankframework/receivers/ExchangeMailListener.java

```diff
@@ -30,7 +30,7 @@
 import org.frankframework.filesystem.MailListener;
 
 /**
- * Microsoft Exchange Implementation of a {@link MailListener}.
+ * Microsoft Exchange Implementation of a {@link ImapListener}.
```

Comment:
This doesn't look correct?

Reply:
Clicking on the link results in a "Error loading Frank!Element" page (https://frankdoc.frankframework.org/#/All/ExchangeMailListener). But I see that it is not entirely true that this pipe is an implementation of an ImapListener.

Reply:
It is indeed not an ImapListener. The error is because no Frank!Doc is generated for the MailListener class, but since it's a `{@link ...}` there is still a link.

I think that it doesn't make sense for the Frank!Doc to have documentation pages for abstract classes that Frank! developers cannot use in their configurations, so it is probably best to just rephrase this.

```suggestion
 * Microsoft Exchange Implementation of a mail filesystem.
```

Reply:
Agreed. The suggestion seems good to me